readme.md

```diff
@@ -1,6 +1,6 @@
 # Frontend practice with catalog page and hovers
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog_hovers/)
+- [DEMO LINK](https://rimidbit.github.io/layout_catalog_hovers/)
```

Comment:
It will be better if you copy this whole line into the description PR, and not just the link